### DISCUSSION TITLE
The overall volume bar is in a inefficient position

### DISCUSSION THREAD

**Discussion by rook747** [File-New-Project/EarTrumpet#1375] [Ideas]
I mostly use the top most volume bar because that Windows 11 volume bar loads very slow and sometimes is grayed out even,
even though I have a new gaming PC with a high end processor. Some of the actions of Microsoft are infuriating.
Anyway, every time an app is opened it's added to the list, so when I want to change overall volume that volume bar is in a different position, higher every time. This is annoying and inefficient. it would be better it the overall volume bar would be at the bottom so that it's always in the same position. I change the volume very fast, it's annoying having to look where it's positioned every time,
and at the bottom is closer to where you click to open it, so it would be faster to reach and more efficient to have it at the bottom.
Either that or have an option to only see the overall volume bar, but having it at the bottom would be better.

Changing the volume also triggers a Windows sound.
This is annoying if i'm watching a video or listening to music and have that sound in between there.

**Comment by TwoCables** [comment]
To prevent sounds from playing while listening to something or watching something, go to **Settings > System > Sound > More sound settings**. When you click "More sound settings", a little dialog box comes up with all of your sound devices. Select the one you're using, click "Properties" (or just double-click the device you're using). In the Properties box that comes up, switch to the 'Advanced' tab and enable the two settings under 'Exclusive Mode".

What would you say to mapping some of your keyboard keys to be Volume Up, Volume Down and maybe one for Mute too?

You can also just simply scroll up and down over the Windows volume icon. You'll see the change represented in the tool tip. So, you don't have to left-click it first.

**Comment by riverar** [comment]
There are multiple suggestions here.

> it would be better it the overall volume bar would be at the bottom [...]

We believe more of our users prefer the apps be closer to the bottom. I don't think we'll be changing this at this time.

> What would you say to mapping some of your keyboard keys to be Volume Up, Volume Down and maybe one for Mute too?

The development builds of EarTrumpet include EarTrumpet Actions which let you do this and more.

> You can also just simply scroll up and down over the Windows volume icon. You'll see the change represented in the tool tip. So, you don't have to left-click it first.

We supported this first a long time ago on Windows 10 ;)

**Reply by rook747** [reply]
I think good programming could give people the option to choose where the overall volume bar is,
now it's in a random position, for me this is chaotic. You said more people like it this way
so I guess more people like randomness and chaos.
I'm leaving Windows 11 anyway until they can get basic things like idi*ticly slow loading volume bars fixed.
I can't find any alternatives for the volume bar that I like + several other problems with Windows 11 that 10 doesn't have
are annoying and some are very disturbing, the path they are taking.

**Reply by riverar** [reply]
Understood. I'll look into possibly putting a toggle into Community Settings.

**Comment by TwoCables** [comment]
> We supported this first a long time ago on Windows 10 ;)

Not for me. When I scroll over the EarTrumpet icon, it just mutes and unmutes.

**Reply by riverar** [reply]
Strange, can you file an issue and fill out that template? Also provide some info on your input (e.g., trackpad or mouse or touch) and OS version/build.

**Comment by TwoCables** [comment]
> Strange, can you file an issue and fill out that template? Also provide some info on your input (e.g., trackpad or mouse or touch) and OS version/build.

The culprit is SmoothScroll: https://www.smoothscroll.net/

Scrolling over the Windows volume icon works.

**Reply by riverar** [reply]
Thanks, filed a bug! https://github.com/File-New-Project/EarTrumpet/issues/1399
